__tests__/utils.test.js

```diff
@@ -1,57 +1,71 @@
-const { stringToDeviceIDType, DeviceIdType } = require("../Utils.js");
+const { intToDeviceIDType, DeviceIdType } = require("../Utils.js");
 
```

Comment:
lets also change the name of the test file to something more specific. lets not put all utils test to same file

Reply:
intToDeviceIDType.test.js?